internal/provider/resource_wlan.go

```diff
@@ -137,6 +138,18 @@ func resourceWLAN() *schema.Resource {
 				Optional:    true,
 				Default:     true,
 			},
+			"minimum_data_rate_2g": {
+				Description:  "Enable minimum data rate control for 2G devices",
```

Comment:
A minor point but I would drop the word "enable" here as, to me at least, it seems to suggest that this is on or off.